bindinate/Makefile.am.template

```diff
@@ -9,13 +9,12 @@ GLUEDIR = $(srcdir)/glue
 SRC_FILES_XML = $(srcdir)/src_files.xml
 CSPROJ_XSLT = $(srcdir)/csproj.xslt
 GEN_SOURCES = $(srcdir)/generated.sources
+CUSTOM_SOURCES = $(srcdir)/custom.sources
 
 gapidir = $(GAPIXMLDIR)
 gapi_DATA = $(API)
 
-# Add any extra source files you need here
-sources =
-
+sources = $(shell grep -Ev '\#.*$$' $(CUSTOM_SOURCES))
```

Comment:
That is definitely a possibility. However, I wanted the custom.sources file only to be a list of source files (without the variable assignment). Is that possible with makefile inclusion?